mbuild/tests/base_test.py

```diff
@@ -123,10 +123,9 @@ def sixpoints(self):
         return molecule
 
     @pytest.fixture
-    def rigid_benzene(self):
+    def benzene(self):
```

Comment:
I think there's already a benzene in line 27. I would remove that once, as it is from smiles, and keep this mol2 version.

Reply:
yeah, this is because RDKIT is what we're using for the from_smiles method, and there is some stochastic nature involved in getting the atomic positions. I think since rdkit is more interested in the chemical connectivity and happy to get a reasonable ground state, but that ground state could be rotated in different ways. Although smiles are really easy to load up for mBuild tests, that one property means that things tend to be less flaky if we just load from a mol2 file.

Reply:
Yeah that's true, but unless I'm mistaken, I don't think that specific test (`test_resnames_parmed_cg`) depends on having consistent atomic positions. It's just creating coarse grain compounds from atomistic compounds.

Reply:
Oh so you're saying it's not flaky, it just fails for the mol2 benzene and passes for the smiles benzene? In that case,I think it's the hierarchy, since that can certainly differ based on the information available in those two different formats.

Reply:
Yeah, this doesn't seem to be a flaky test, but related to how hierarchy is set.